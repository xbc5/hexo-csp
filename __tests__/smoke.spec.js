const getSandbox = require("./support/sandbox");
const template = require("./helpers/template");
const { process, contentFor } = require("hexo-test-utils");
const sandbox = getSandbox();

describe("using the hexo-test-utils", () => {
  it("should return a rendered document", async () => {
    const ctx = await sandbox({ fixtureName: "default" });
    await process(ctx);
    const content = await contentFor(ctx, "fake.html");

    expect(content.toString().trim()).toBe(template(ctx));
  });
});
