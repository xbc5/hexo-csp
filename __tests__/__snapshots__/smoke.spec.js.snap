// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`using the hexo-test-utils should return a rendered document 1`] = `
"<!DOCTYPE html>
<html lang=\\"en\\">
  <head>
    <title>Fake Title</title>
  </head>
  <body></body>
</html>"
`;
